src/DssSpell.sol

```diff
@@ -75,334 +60,240 @@ contract DssSpellAction is DssAction {
-
-        // Set RWA007-A Debt Ceiling to 0
-        DssExecLib.setIlkDebtCeiling("RWA007-A", 0);
-
-        // Initiate RWA007-A soft liquidation by calling `tell()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).tell("RWA007-A");
-
-        // Write-off the debt of RWA007-A and set its oracle price to 0 by calling `cull()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).cull("RWA007-A", RWA007_A_URN);
-
-        // Note: update the spot value in vat by propagating the price
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // Note: in order to decrease global debt ceiling, we need to fetch current `line`
-        (,,, uint256 line2,) = VatAbstract(MCD_VAT).ilks("RWA014-A");
-
-        // Reduce RWA014-A Debt Ceiling by 1.5 billion Dai from 1.5 billion Dai to 0
-        DssExecLib.setIlkDebtCeiling("RWA014-A", 0);
-
-        // Initiate RWA014-A soft liquidation by calling `tell()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).tell("RWA014-A");
-
-        // Write-off the debt of RWA014-A and set its oracle price to 0 by calling `cull()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).cull("RWA014-A", RWA014_A_URN);
-
-        // Note: update the spot value in vat by propagating the price
-        DssExecLib.updateCollateralPrice("RWA014-A");
-
-        // Note: decrease global line
-        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() - (line1 + line2));
-
-        // ---------- Pinwheel DAO Resolution ----------
-        // Forum: https://forum.sky.money/t/coinbase-web3-wallet-legal-overview/24577/3
-
-        // Approve DAO Resolution at QmYJUvw5xbAJmJknG2xUKDLe424JSTWQQhbJCnucRRjUv7
-        // Note: see `dao_resolutions` public variable declared above
-
-        // ---------- AAVE Revenue Share Payment ----------
-        // Forum: https://forum.sky.money/t/spark-aave-revenue-share-calculation-payment-5-q3-2024/25286
-
-        // AAVE Revenue Share - 234089 DAI - 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c
-        DssExecLib.sendPaymentFromSurplusBuffer(AAVE_V3_TREASURY, 234_089);
-
-        // ---------- Spark Spell ----------
-        // Forum: https://forum.sky.money/t/oct-3-2024-proposed-changes-to-spark-for-upcoming-spell/25293
-        // Poll: https://vote.makerdao.com/polling/QmbHaA2G
-        // Poll: https://vote.makerdao.com/polling/QmShWccA
-        // Poll: https://vote.makerdao.com/polling/QmTksxrr
+        // ---------- Init Base Native Bridge ----------
+        // Forum: https://forum.sky.money/t/spell-contents-2024-10-31/25421/
+        //
+        // Set Escrow contract for L1 Bridge
+        // Register USDS, sUSDS on L1 Bridge
+        // Give max approval on Enscrow contract for L1 Bridge (USDS, sUSDS tokens)
+        // Execute L2 Bridge spell through Gov Relay (register and set maxWithdrawals for USDS, sUSDS tokens on L2 Bridge)
+        // Set BASE_GOV_RELAY, BASE_ENSCROW, BASE_TOKEN_BRIDGE and BASE_TOKEN_BRIDGE_IMP to CHAINLOG
```

Comment:
These comments aren't in exec sheet and I don't think they are necessary here.
Also, if the necessary comments are added and not in the exec sheet we should use `note` prefix

Related checklist can be found [here](https://github.com/makerdao/pe-checklists/blob/5208017c2c5e04f9ec03817856e06ff9d0fe9ad1/spell/spell-reviewer-mainnet-checklist.md?plain=1#L24)